web/src/views/SubmissionPortal/Components/ValidateSubmit.vue

```diff
@@ -1,14 +1,14 @@
 <script lang="ts">
 import { defineComponent } from '@vue/composition-api';
 import { writeFile, utils } from 'xlsx';
-import { submit, submitPayload, sampleData } from '../store';
+import { submit, submitPayload } from '../store';
 import { saveAs } from '@/util';
 import useRequest from '@/use/useRequest';
 
 export default defineComponent({
   setup(_, { root }) {
     function downloadSamples() {
-      const worksheet = utils.aoa_to_sheet(sampleData.value);
+      const worksheet = utils.aoa_to_sheet([]); // TODO: sampleData.value);
```

Comment:
D'oh! Forgot about that one. It's not exactly a simple fix, and I punted on it because I think that component isn't used anymore and can just be deleted. I made an issue to follow up on that: https://github.com/microbiomedata/nmdc-server/issues/852